cli/config/version.go

```diff
@@ -31,15 +33,48 @@ Server: Not Configured`, false
 Server: Failed to get the server version - %s`, err.Error()), false
 	}
 
+	versionMatch := false
 	version := meta.GetVersion()
-	isVersionMatch := version == Version
-	if isVersionMatch {
+	if !isSemver(version) || !isSemver(Version) {
```

Comment:
this is the old behaviour. useful for CI and development, where versions are not semantic (like `dev` or some commit sha)